pkg/build/api/types.go

```diff
@@ -424,7 +424,7 @@ type BuildStrategy struct {
 	CustomStrategy *CustomBuildStrategy
 
 	// JenkinsPipelineStrategy holds the parameters to the Jenkins Pipeline build strategy.
-	// This strategy is experimental.
+	// This strategy is a tech preview.
```

Comment:
(pedantic: a feature that is released as a tech preview is, by definition, in the state of tech preview)